docs/guides/user/cli-client.md

```diff
@@ -51,13 +51,15 @@ Options:
 Commands:
   analyze         Analyze timelines.
   config          Configuration for this CLI tool.
-  events          Manage events
+  events          Manage event.
   import          Import timeline.
+  intelligence    Manage intelligence within a sketch.
+  sample-data     Commands for generating sample or test data.
```

Comment:
Do we still need this line when moving this into a separate script?

Reply:
uh true that should be removed